src/main/webapp/js/navigation-loader.js

```diff
@@ -86,24 +85,29 @@ function addLoginOrLogoutLinkToNavigation() {
         );
 
         navigationElement.appendChild(
-          createListItem(createLink('/feed.html', 'Public Feeds'))
+          createListItem(createLink('/see-around.html', 'See Around'))
         );
 
         navigationElement.appendChild(
-          createListItem(createLink('/map.html', 'Map'))
+          createListItem(createLink('/one-glance.html', 'One Glance'))
         );
 
         navigationElement.appendChild(
-          createListItem(createLink('/stats.html', 'Statistics'))
+          createListItem(createLink('https://www.google.com/maps/search/google+map/', 'Map'))
         );
 
         navigationElement.appendChild(
-          createListItem(createLink('/community.html', 'Community'))
+          createListItem(createLink('/bugs-issues', 'Bugs'))
         );
 
         navigationElement.appendChild(
           createListItem(createLink('/logout', 'Logout'))
         );
+
+        // Probably this is not needed in our prototype
+        // navigationElement.appendChild(
+        //   createListItem(createLink('/feed.html', 'Public Feeds'))
```

Comment:
Is it Public Feeds is that main feature where users can see everyone's messages?